manager/templates/default/element/tv/renders/input/url.tpl

```diff
@@ -1,37 +1,48 @@
 <select id="tv{$tv->id}_prefix" name="tv{$tv->id}_prefix" onchange="MODx.fireResourceFormChange();">
-{foreach from=$urls item=url}
-	<option value="{$url}" {if $url == $selected|default}selected="selected"{/if}>{$url}</option>
-{/foreach}
+    {foreach from=$urls item=url}
+        <option value="{$url}" {if $url == $selected|default}selected="selected"{/if}>{$url}</option>
+    {/foreach}
 </select>
+
 <input id="tv{$tv->id}" name="tv{$tv->id}"
-	type="text"
-	value="{$tv->get('processedValue')}"
-	onchange="MODx.fireResourceFormChange();"
-	class="textfield x-form-text x-form-field"
-	style="width: 283px;"
+    type="text"
+    value="{$tv->get('processedValue')}"
+    onchange="MODx.fireResourceFormChange();"
+    class="textfield x-form-text x-form-field"
+    style="width: 283px;"
 />
+
 <script type="text/javascript">
 // <![CDATA[
 Ext.onReady(function() {
-	MODx.makeDroppable(Ext.get('tv{$tv->id}'));
-
+    MODx.makeDroppable(Ext.get('tv{$tv->id}'));
     var fld = MODx.load({
         xtype: 'combo'
         ,transform: 'tv{$tv->id}_prefix'
         ,id: 'tv{$tv->id}_prefix'
         ,triggerAction: 'all'
         ,width: 100
-        ,allowBlank: false
         ,maxHeight: 300
         ,typeAhead: false
         ,forceSelection: false
+        ,allowBlank: true
         ,msgTarget: 'under'
         ,listeners: { 'select': { fn:MODx.fireResourceFormChange, scope:this}}
     });
+    fld.wrap.applyStyles({
+        display: "inline-block"
+    });
 
-	fld.wrap.applyStyles({
-		display: "inline-block"
-	});
+    var fld = MODx.load({
```

Comment:
Is this addition really needed?

Reply:
Well, this is a text field handler, it is for him that you need `allowBlank: false`, because links can be without the selected type in the drop-down list.
Or did I misunderstand you?